src/components/MedicationReports.js

```diff
@@ -30,10 +30,17 @@ const MedicationReports = observer(({ assembly }) => (
       .participant_account
       .information
       .medication_reports
+      .sort(function(a,b){
+        return new Date(b.timestamp) - new Date(a.timestamp);
+      })
       .map(({timestamp, id, took_medication, not_taking_medication_reason}) => (
-        // TODO: Sort by date and link reports
         <Answer>
-          <Time key={id}>{DateTime.fromISO(timestamp).toLocaleString(DateTime.DATETIME_SHORT)}</Time>
+          <Time key={id}>
+            {DateTime
+              .fromISO(timestamp, {zone: 'utc'})
+              .setLocale(assembly.locale)
+              .toLocaleString(DateTime.DATETIME_SHORT)}
```

Comment:
This looks like perfect treatment for `timestamp`.

Reply:
Thanks it was frustrating for a while, `{zone: 'utc'}` is a good thing to remember moving forward.